tests/sentry/integrations/utils/test_code_mapping.py

```diff
@@ -26,10 +26,6 @@
 
 UNSUPPORTED_FRAME_FILENAMES = [
     "async https://s1.sentry-cdn.com/_static/dist/sentry/entrypoints/app.js",
-    "/_static/dist/sentry/entrypoints/app.js",
-    "/node_modules/@wix/wix-one-app-rich-content/src/rich-content-native/plugin-video/video.js",  # 3rd party
-    "/_static/dist/sentry/entrypoints/app.js",
-    "/_static/dist/sentry/chunks/node_modules_emotion_unitless_dist_emotion-unitless_esm_js-app_bootstrap_initializeMain_tsx-n-f02164.80704e8e56b0b331c4e2.js",
```

Comment:
Removing these from `UNSUPPORTED_FRAME_FILENAMES` only means that we will not raise an exception when trying to create an object of `FrameFilename`.